app/models/concerns/geojson_formattable.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module GeojsonFormattable
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def format_geojson_epsg(column_name)
```

Comment:
the map component introduces a breaking change here - https://github.com/theopensystemslab/map/pull/256/files

This is to ensure compatibility with the previous way we store geojson